src/org/opendatakit/briefcase/ui/export/ExportPanelForm.java

```diff
@@ -41,8 +50,12 @@
   private JPanel rightActions;
   private JButton selectAllButton;
   private JButton clearAllButton;
+  private JLabel lblExporting;
```

Comment:
I forgot this one before, sorry. Rename the field to `exportingLabel` for conformity with other fields.